packs/chatops_tests/actions/test_aliases_with_slack.py

```diff
@@ -175,12 +181,12 @@ def test_help_longcut(self):
             if filtered_messages:
                 messages.extend(filtered_messages)
 
-        self.assertEqual(1, len(messages))
-        if len(messages) != 1:
+        self.assertEqual(2, len(messages))
+        if len(messages) != 2:
             time.sleep(self.WAIT_FOR_MESSAGES_TIMEOUT)
 
         # Help commands don't get acked
-        self.assertIn("!help - Displays all of the help commands that this bot knows about.", messages[0]['text'])
+        self.assertIn("!help - Displays all of the help commands that this bot knows about.", messages[1]['text'])
```

Comment:
Same here - pass in `combined_text` instead of using the text from a specific message.